HealthcareIS.sln

```diff
@@ -9,7 +9,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WPFHospitalEditor", "WPFHos
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HospitalWebApp", "HospitalWebApp\HospitalWebApp.csproj", "{7CB35A4E-A104-4098-A2FE-23E4A42856B0}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HealthcareBaseTests", "HealthcareBaseTests\HealthcareBaseTests.csproj", "{3EF55118-B41F-4E4F-BB2C-07BD77A46FCE}"
+<<<<<<< HEAD
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HealthcareBaseTests", "HealthcareBaseTests\HealthcareBaseTests.csproj", "{9564E34A-6EF5-4E4E-A250-2A348C5A031D}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HospitalWebAppTests", "HospitalWebAppTests\HospitalWebAppTests.csproj", "{51B18CC7-B280-4A8F-A019-CAE492B7ACDF}"
+=======
 EndProject
```

Comment:
Why is this in PR?

Reply:
I don't know, haven't noticed it because it wasn't causing any errors. Probably I forgot to resolve it.